guides/common/modules/proc_upgrading-project-in-place-using-leapp.adoc

```diff
@@ -12,8 +12,16 @@ endif::[]
 ifdef::satellite[]
 * Review Known Issues before you begin an upgrade.
 For more information, see {ReleaseNotesURL}ref_known-issues_assembly_introducing-red-hat-satellite[Known Issues in {ProjectName} {ProjectVersion}].
+* If you run {Project} in a disconnected environment, ensure it meets the following prerequisites:
```

Comment:
I think you could move this to an extra block ".Prerequisites for Disconnected Environments" and then keep a "flat" list structure.